roxie/ccd/ccdqueue.cpp

```diff
@@ -876,7 +876,7 @@ void AgentContextLogger::set(ISerializedRoxieQueryPacket *packet)
         if (intercept || mergeAgentStatistics)
         {
             RoxiePacketHeader newHeader(header, ROXIE_TRACEINFO, 0);  // subchannel not relevant
-            output.setown(ROQ->createOutputStream(newHeader, true, *this));
+            output.setown(ROQ->createOutputStream(newHeader, false, *this));
```

Comment:
reason for this change?  To ensure the stats get serialized?

Reply:
It makes it more likely that stats from agents reach server before activity stops listening for them

Reply:
Plus it means the stats are not sent higher priority than data